safe-global/web-core
# style: new theme

## What it solves

Update light/dark themes to match new style.

## How this PR fixes it

The colour palettes have been updated according to Figma and components adjusted accordingly.

Note: the illustration updates will follow in another PR.

## How to test it

Open the Safe in light/dark mode and observe the new colour scheme. The design should match [Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=1%3A3).
